package.json

```diff
@@ -63,19 +63,19 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s"
   },
   "dependencies": {
-    "@stylable/dom-test-kit": "^1.1.0",
     "classnames": "^2.2.6",
     "lodash": "^4.17.11",
     "react-resize-detector": "^4.1.3",
-    "wix-ui-core": "^2.0.889",
-    "wix-ui-icons-common": "^2.0.0"
+    "wix-ui-core": "^3.0.0",
+    "wix-ui-icons-common": "^2.0.0",
+    "yoshi-style-dependencies": "^4.6.0"
   },
   "devDependencies": {
     "@applitools/eyes-storybook": "^2.7.28",
     "@storybook/addon-links": "^5.0.9",
     "@storybook/react": "^5.0.9",
-    "@stylable/cli": "^1.0.0",
-    "@stylable/core": "^1.1.0",
```

Comment:
We're using `@stylable/core` here: https://github.com/wix/wix-ui-tpa/blob/87648ef57f67d662dc7ce9d8a0de5109245ee914/.storybook/stylable-metadata-plugin.js#L9
I'm wondering if it's ok to remove it.

But since I can see the storybook is generated and working good, I can only assume that it's ok 🤷‍♂️

Reply:
wix-ui-tpa@1.10.2 /Users/nitzansi/Projects/wix-ui-tpa
├─┬ @stylable/cli@3.5.2
│ ├── @stylable/core@3.5.2 
│ ├─┬ @stylable/node@3.5.2
│ │ ├── @stylable/core@3.5.2  deduped
│ │ └─┬ @stylable/module-utils@3.5.2
│ │   └── @stylable/core@3.5.2  deduped
│ └─┬ @stylable/optimizer@3.5.2
│   └── @stylable/core@3.5.2  deduped
├─┬ @stylable/dom-test-kit@3.5.2
│ └── @stylable/core@3.5.2  deduped
├─┬ @ui-autotools/a11y@2.7.3
│ └─┬ @ui-autotools/node-utils@1.5.3
│   └─┬ @stylable/node@1.4.0
│     └── @stylable/core@1.4.0 
├─┬ wix-ui-core@3.0.34
│ └─┬ yoshi-stylable-dependencies@4.78.0
│   ├─┬ @stylable/jest@3.5.0
│   │ └── @stylable/core@3.5.2  deduped
│   ├─┬ @stylable/module-utils@3.5.0
│   │ └── @stylable/core@3.5.2  deduped
│   └─┬ @stylable/node@3.5.1
│     └── @stylable/core@3.5.2  deduped
└─┬ yoshi@4.80.3
  └─┬ yoshi-common@4.80.3
    ├── @stylable/core@1.4.0 
    └─┬ @stylable/node@1.4.0
      └── @stylable/core@1.4.0  deduped

Reply:
This is the npm ls tree for @stylable/core.
To be honest, I'm not sure if we need to install it explicitly because so many packages are relying on it. 
Maybe we should do it to prevent versions mess

Reply:
Yeah, I guess we can leave it, because it's working ok.

Reply:
I mean, we can leave the PR as it is now

Reply:
this reminds me, that `@ui-autotools` should probably also be upgraded